src/classes/path-utils.ts

```diff
@@ -18,15 +18,21 @@ const defaultProjectPath = "~/.meta-proj-cli/projects";
 export class PathUtils {
 
   public static savePath = async () : Promise<string> => {
-    return await PathUtils.translatePath(defaultSavePath);
+    let os : string;
+    await OsUtils.getOS() ? os = await OsUtils.getOS() : os = OsUtils.detectOS();
+    return await PathUtils.translatePath(defaultSavePath, os);
```

Comment:
this same set of code is repeated multiple times, make into function and call instead